.github/ISSUE_TEMPLATE.md

```diff
@@ -5,12 +5,13 @@ searching the issues list. If there is a duplicate, re-open or add a comment to
 existing issue instead of creating a new one. If you are reporting a bug,
 make sure to include relevant information asked below to help with debugging.
 
-## GENERAL HELP QUESTIONS ##
-Github Issues is for bug reports and feature requests. If you have general support
-questions, the following locations are a good place:
+# Need general help?
+
+GitHub issues are for bug reports and feature requests. If you have general support
+questions, the following resources are a good place to start:
 
 - Slack channel (#samdev): https://awssamopensource.splashthat.com/
-- Post a question in StackOverflow  with "aws-sam-cli" tag
+- Post a question on Stack Overflow with the tag `aws-sam-local`
```

Comment:
Is there a reason for switching this back to `aws-sam-local`. This should match the current project name. Can you update this back to `aws-sam-cli`, or explain why we should keep the `*-local`?

Reply:
Hmm.. Looks like we need to update that on SO, but not sure how to be honest. Let me see if I can't figure that out. If that is a long process, we can address that later. :)

Reply:
@jfuss Sounds like a plan :) I opened a discussion on StackOverflow Meta to have it renamed -- will keep you posted.

Reply:
@rbrtmrtn  How did the discussion in StackOverflow go?

Reply:
I got one upvote... :)

https://meta.stackoverflow.com/questions/368189/retag-aws-sam-local-to-aws-sam-cli

I'm familiar enough with SO admin stuff to know if there's some way to bump the issue. Anyone else?